.size-snapshot.json

```diff
@@ -1,31 +0,0 @@
-{
```

Comment:
:(

Reply:
In think it's just not ready yet, I guess it wasn't intentional to remove it

Reply:
Yea, I wasn’t able yet to make this work again.

Reply:
I remember tsdx added or wanted to add support for extensible config.